packages/app/lib/src/features/home/data/post_repository.dart

```diff
@@ -77,9 +77,10 @@ final class _AppwritePostRepository implements PostRepository {
           LocalFeed() => [
               Query.between('lat', lat - 2, lat + 2),
               Query.between('lng', lng - 2, lng + 2),
+              Query.orderDesc('timestamp'),
             ],
           WorldFeed() => [
-              // No filter.
+              Query.orderDesc('timestamp'),
```

Comment:
If this is the same either way, can we move this out of conditional? Both for perf and think-about-ability.

Reply:
added